### PR TITLE
Automate fund-round creation

### PR DESCRIPTION
**Pull request by albertkol**
Automates the whole setting up Apply and Assess to use a FAB created fund, round and form. 
Lots of our code is moved at the moment, so this will soon break, but if there is any value in this script, we will update it.

Pre-requisites:
1. Be checked out on `main` on the following repos, pull latest changes, stash uncommitted changes. The script will create a new branch on multiple repos with the required config.
2. You can either create your own FAB fund/round/form or ask for a sample; you need to extract the config from FAB and paste it in `/fund-round/.`. 
3. You might need to clean up a bit FAB's exported stuff. Name the dir nicely without special chars/spaces.

Optional step:
Do a `make down`. Keep things clean.

### How to test
1. Make sure your FAB directory with the `html`, `fund-store` and `form-runner` config are inside `/fund-round`
2. Run `./scripts/fund-round-creator.sh` and follow the prompt
3. Until [PR](https://github.com/communitiesuk/funding-service-pre-award-stores/pull/13) is merged you will have to run seed the fund manually: 
```bash
docker exec funding-service-pre-award-stores-1 python -m fund_store.scripts.fund_round_loaders.load_fund_round_from_fab --fund_short_code <fund-short-name-uppercase>
```
4. `make up`

You can now go to:
https://frontend.levellingup.gov.localhost:3008/funding-round/fund-short-name/round-short-name
Create a quick application.

You can now go to:
https://assessment.levellingup.gov.localhost:3010
You can see you application there.

### CODE REVIEW

File: scripts/fund-round-creator.sh [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+#   Main script
+###############################################################################
+
+# List all directories inside the fund-round directory
+print_header "Available directories in $FUND_ROUND_DIR:"
+dirs=()
+i=1
+for dir in "$FUND_ROUND_DIR"/*; do
+    if [[ -d "$dir" ]]; then
+        dirs+=("$dir")
```
Comment (samuelhwilliams) [inline scripts/fund-round-creator.sh:96]:
```suggestion
for dir in $(find $FUND_ROUND_DIR -type d -depth 1); do
    dirs+=("$dir")
```

Comment (albertkol) [reply scripts/fund-round-creator.sh:96]: Your suggestion didn't work. Using `find`, this worked instead:
`find $FUND_ROUND_DIR -mindepth 1 -maxdepth 1 -type d | sed 's|.*/||'`

Comment (samuelhwilliams) [reply scripts/fund-round-creator.sh:96]: Yeah ok compat issues maybe. No biggie.

File: scripts/fund-round-creator.sh [hunk 2]
```diff
@@ -111,7 +111,7 @@ print_message "fund_store file: $FUND_STORE_FILE"
 print_prompt "Did you ruff formatted the form_runner? (1/0): "
 read -p "" confirm
 
-if [[ "$confirm" == "1" ]]; then
+if [[ "$confirm" == "0" ]]; then
     print_error "Exiting script. Come back after you ruff format the form_runner."
     exit 1
 fi
```
Comment (samuelhwilliams) [inline scripts/fund-round-creator.sh:117]: Why not just ruff format it for them? 👀  `uv tool run ruff format <dir>` should do the trick (sad that this is needed 😭)

Comment (albertkol) [reply scripts/fund-round-creator.sh:117]: :thinking: I thought a bit of ways to automate the ruff-ing. Will give it a throught..

Comment (samuelhwilliams) [reply scripts/fund-round-creator.sh:117]: even better idea, don't dynamically write python files when we're just exporting data 😬

**Review by samuelhwilliams — APPROVED** [review]
it "works"? ✅

**Pull request merged**